OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -110,7 +110,7 @@ public override bool Tick(Actor self)
 
 			if (ShouldLandAtBuilding(self, dest))
 			{
-				var exit = dest.FirstExitOrDefault();
+				var exit = dest.FirstExitOrDefault(null, null, self.CenterPosition);
```

Comment:
I think it would be better to use a named argument for `nearestTo` here, instead of `null, null`

Reply:
`nearestTo` is the last argument so naming it wouldn't change the `null, null`.
Instead it'd be `null, null, nearestTo: self.CenterPosition`.
Perhaps I'm not understanding what you're suggesting.

Reply:
No, it would be `dest.FirstExitOrDefault(nearestTo: self.CenterPosition)`

Using named arguments means the order doesn't matter and arguments can be skipped as long as the skipped arguments have a default value, which they have.